frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectFeatureToggles.tsx

```diff
@@ -86,10 +86,11 @@ export const ProjectFeatureToggles = ({
         createdAt: tableState.createdAt,
     };
 
+    const { columns: _, ...apiTableState } = tableState;
```

Comment:
omit columns and rename to _ to avoid name clash with the existing columns variable